doc/tutorials/linear_regression/linear_regression.txt

```diff
@@ -141,7 +142,67 @@ dataset is one dimensional, and the last column has the \f$y\f$ values, or
 responses, for each row. You can specify these responses in a separate file if
 you want, using the \c --input_responses, or \c -r, option.
 
-@subsection cli_ex2_lrtut Compute model and predict at the same time
+@subsection cli_ex2_lrtut Example Multivariate Linear regression
+
+Multivariate linear regression involves more than just one variable coefficients, in this example
+we will try to fit the data which contain four variables from \c dataset_2.csv using our executables.
+
+@code 
+$ mlpack_linear_regression --training_file dataset_2.csv -v -M lr.xml
+[INFO ] Loading 'dataset_2.csv' as CSV data.  Size is 5 x 5.
+[INFO ] 
+[INFO ] Execution parameters:
+[INFO ]   help: 0
+[INFO ]   info: 
+[INFO ]   input_model_file: 
+[INFO ]   lambda: 0
+[INFO ]   output_model_file: lr.xml
+[INFO ]   output_predictions_file: 
+[INFO ]   test_file: 
+[INFO ]   training_file: dataset_2.csv
+[INFO ]   training_responses_file: 
+[INFO ]   verbose: 1
+[INFO ]   version: 0
+[INFO ] Program timers:
+[INFO ]   load_regressors: 0.000060s
+[INFO ]   loading_data: 0.000050s
+[INFO ]   regression: 0.000049s
+[INFO ]   total_time: 0.000118s
+
+$ cat dataset_2.csv
+0,0,0,0,14
+1,1,1,1,24
+2,1,0,2,27
+1,2,2,2,32
+-1,-3,0,2,17
```

Comment:
It could be useful to put a little text between these sections so that a reader can know what they are, and that they are distinct---right now it's all in one code block, which could make it all seem like it's one thing.